### PR TITLE
Multiple fallback languages and region support

### PR DESCRIPTION
**Pull request by git-no**
Simple approach via one translation file. Fallback use YAML standard methods.
- Lib enhancements included
- Example included
- Documentation fix included

### CODE REVIEW

**Comment by Anthony-Gaudino** [comment]
Hi @git-no,

As I could understand you want to have a fallback to another language other than the default one and be able to use a single file to add all your translations.

--------

Could you please explain about your needs and how do you use those features.

Do you need fallback of full language to another language and also fallback of parts of a language to another?

**Comment by git-no** [comment]
#### Starting position:
- Having a website target to audience in multiple regions or languages.

#### Goal:
- Get hight rankend in search machines in every target country, having a good SEO .
- As less manually work as possible.

#### Requirement:
- Jekyll support of multiple regions and languages.

#### Situation:
- Google prefers and ranks websites higher with region country (en-UK) than with global languages (en).

Simple approach:
```
<html lang="en">
http://www.domain.com/en/
```

Better SEO approach:
```
<html lang="en-UK">
http://www.domain.com/enuk/

<html lang="en-CA">
http://www.domain.com/enca/
```

#### Solution:
Serving websites dedicated to region and language instead of just language.

Maintaining language files with flexible fallback. This is possible just with YAML, no ruby addon required.

Look at my language.yaml example file.

File: README.md [hunk 1]
```diff
@@ -272,7 +272,7 @@ titles:
 This plugin gives you the variables
 
 ```liquid
-{{ page.lang }}
+{{ site.lang }}
```
Comment (joshcummingsdesign) [inline README.md]: This should be merged

**Comment by joshcummingsdesign** [comment]
@kurtsson #84 fixes README.md issue. @git-no Can you resolve the conflicts and we can then re-review?

**Comment by joshcummingsdesign** [comment]
@kurtsson This can be closed. @git-no Happy to review a new PR once the conflicts have been resolved.

**Pull request closed without merging**